package.json

```diff
@@ -7,7 +7,10 @@
     "node": ">=22.0.0"
   },
   "scripts": {
+    "fetch-limits": "node scripts/fetch-network-limits.mjs",
+    "predev": "pnpm fetch-limits",
     "dev": "NEXT_PUBLIC_COMMIT_HASH=$(git rev-parse --short HEAD) NEXT_PUBLIC_ENABLE_EXPLORER=true next dev",
+    "prebuild": "pnpm fetch-limits",
```

Comment:
`predev` and `prebuild` scripts run automatically before `dev` and `build` respectively (npm feature).